lib/documentLoaders/node.js

```diff
@@ -114,13 +114,15 @@ module.exports = ({
       alternate = linkHeaders['alternate'];
       if(alternate &&
         alternate.type == 'application/ld+json' &&
+        // eslint-disable-next-line max-len
         !(res.headers['content-type'] || '').match(/^application\/(\w*\+)?json$/)) {
```

Comment:
I'd prefer a line break at `.match` to keep the style rather than a style exception.

```suggestion
        !(res.headers['content-type'] || '')
          .match(/^application\/(\w*\+)?json$/)) {
```